### PR TITLE
Add AWS release 10.1.1

### PR DESCRIPTION
**Pull request by rossf7**
Fixes https://github.com/giantswarm/giantswarm/issues/8328

### CODE REVIEW

File: release-notes/aws/v10.1.1.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# :zap: Giant Swarm Release 10.1.1 for AWS is now active for you! :zap:
```
Comment (rossf7) [inline release-notes/aws/v10.1.1.md]: @cokiengchiara This is what I have so far for the release notes.

cc @puja108

**Comment by omegas27** [comment]
LGTM.

File: release-notes/aws/v10.1.1.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+This Node Pools release fixes a problem with CoreDNS in clusters using custom
+cluster IP ranges. Additionally CPU limits have been removed from multiple
+default apps.
```
Comment (puja108) [inline release-notes/aws/v10.1.1.md]:
```suggestion
default apps to increase general reliability of these.
```

Comment (puja108) [reply release-notes/aws/v10.1.1.md]: Could add a link to why that is the case. WDYT?

Comment (rossf7) [reply release-notes/aws/v10.1.1.md]: Yes good idea. @sslavic Do you have a link for this?

**Comment by rossf7** [comment]
Going with this so I can add the new WIP releases.

@puja108 @sslavic If you have a good link for why the CPU limits were removed please post it and I'll do a follow up PR.

**Pull request merged**